sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -49,6 +49,15 @@ export interface ServiceBusSender {
    * - All messages passed to the same sendMessages() call should have the same `sessionId` (if using
    *  sessions) and the same `partitionKey` (if using partitions).
    *
+   * **Note:**
+   *
+   *    __If you want to send messages of size greater than 1MB, please send individual messages instead of sending a batched message or an array of messages like below.__
```

Comment:
We can also link out to the SB doc that states the limitation https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-premium-messaging#large-messages-support